src/main/webapp/lounge.html

```diff
@@ -3,24 +3,25 @@
   <head>
     <meta charset="UTF-8">
     <title>YouTube Lounge</title>
-    <link rel="shortcut icon" href="images/sofa.png" type="image/x-icon">
-    <link rel="icon" href="images/sofa.png" type="image/x-icon">
+    <link rel="shortcut icon" href="../images/sofa.png" type="image/x-icon">
+    <link rel="icon" href="../images/sofa.png" type="image/x-icon">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <link rel="stylesheet" href="https://code.getmdl.io/1.1.3/material.orange-indigo.min.css">
-    <link rel="stylesheet" href="chat_style.css"></link>
-    <link rel="stylesheet" href="index_style.css"></link>
-    <link rel="stylesheet" href="queue_style.css"></link>
+    <link rel="stylesheet" href="../chat_style.css"></link>
+    <link rel="stylesheet" href="../index_style.css"></link>
+    <link rel="stylesheet" href="../queue_style.css"></link>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&amp;lang=en">
     <script defer src="https://code.getmdl.io/1.1.3/material.min.js"></script>
-    <script src="index_script.js"></script>
-    <script src="config.js"></script>
+    <script src="../index_script.js"></script>
+    <script src="../config.js"></script>
+    <script src="../IFrame.js"></script>
```

Comment:
I think you only need to add the IFrame script tag you have on the bottom instead of having it twice.

Reply:
Thanks for catching that!